lib/kitchen/transport/winrm.rb

```diff
@@ -118,6 +119,22 @@ def execute(command)
           end
         end
 
+        def retry?(current_try, max_retries, retryable_exit_codes, exception)
+          # Avoid duplicating Kitchen::Transport::Base#retry?
+          result = super
+          return result if result == true
+
+          case exception
+          when WinRM::WinRMHTTPTransportError
+            current_try <= max_retries &&
+              [400, 500].include?(exception.status_code)
+          when WinRM::WinRMWSManFault
+            current_try <= max_retries
+          else
+            false
+          end
```

Comment:
sort of the same thing here, only retain the `case` and `return` out of the branches then just have the last statement be `false` for the default return value.